libethcore/Ethasher.cpp

```diff
@@ -43,6 +43,12 @@ Ethasher* dev::eth::Ethasher::s_this = nullptr;
 bytes const& Ethasher::cache(BlockInfo const& _header)
 {
 	RecursiveGuard l(x_this);
+	if (_header.number > EPOCH_LENGTH*2048) {
```

Comment:
Doesn't comply with coding standards:
- Preprocessor define should contain "namespace" prefix (e.g. `ETHASH_EPOCH_LENGTH`)
- Space on either side of `*`.
- `{` goes on own line.

Whitespace for much of this PR is bad.

See cpp-ethereum/CodingStandards.txt